### PR TITLE
Clean cypher-where-predicate logic

### PR DESCRIPTION
**Pull request by angrykoala**
# Description
This PR cleans a bit the logic regarding creating where predicates

### CODE REVIEW

File: packages/graphql/src/translate/where/create-where-predicate.ts [hunk 1]
```diff
@@ -37,46 +36,25 @@ export function createCypherWherePredicate({
...
 }): CypherBuilder.Predicate | undefined {
-    const mappedProperties = mapPropertiesToOperators({
-        whereInput,
-        targetElement,
-        element,
-        context,
+    const whereFields = Object.entries(whereInput);
+
+    const predicates = whereFields.map(([key, value]): CypherBuilder.Predicate | undefined => {
+        if (key === "OR") {
```
Comment (MacondoExpress) [inline packages/graphql/src/translate/where/create-where-predicate.ts:42]: The function `mapPropertiesToPredicates` could potentially be removed by refactoring the following: 
```suggestion
        if (key === "OR" || key === "AND") {
            const nested = value.map((v) => {
                return createWherePredicate({ whereInput: v, element, targetElement, context });
            }
            if (key === "OR") {
                return CypherBuilder.or(...nested);
            }
            return CypherBuilder.and(...nested);
    });
```

**Review by MacondoExpress — DISMISSED** [review]
LGTM, just left a not blocking suggestion 👍

File: packages/graphql/src/translate/where/create-where-predicate.ts (hunk 1 above)
Comment (angrykoala) [reply packages/graphql/src/translate/where/create-where-predicate.ts]: That's a great point, I'll make this change.
In the last commit, I also further refactored this by extracting the logic inside `if (key === "OR" || key === "AND") {` to a separate function. Let me know what you think of that

Comment (MacondoExpress) [reply packages/graphql/src/translate/where/create-where-predicate.ts]: It looks good to me!

**Pull request merged**